dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -3762,7 +3762,7 @@ def generate_issue_content(
 ):
     from github import Github, Issue, PullRequest, UnknownObjectException
 
-    PullRequestOrIssue = Union[PullRequest.PullRequest, Issue.Issue]
+    PullRequestOrIssue = PullRequest.PullRequest | Issue.Issue
```

Comment:
This is to fix
`Testing: avoid  UP038 Use X | Y in isinstance call instead of (X, Y)`

Reply:
I think we can use pyupgrade to switch to 3.10 syntax where possible :)